randomize_deck_startup.sh

```diff
@@ -51,6 +52,7 @@ replace_css_video() {
   OLD_CSS="video{flex-grow:0;width:300px;height:300px;z-index:10}"
   NEW_CSS="video{flex-grow:1;width:100%;height:100%;z-index:10}  "
```

Comment:
Can also remove this trailing space now, yes?

Reply:
Yep! No need for it.